keras/optimizers/optimizer_experimental/adamw.py

```diff
@@ -215,6 +216,8 @@ def get_config(self):
     })
     return config
 
+  def exclude_from_weight_decay(self, var_list):
+    self._exclude_from_weight_decay = var_list
```

Comment:
Need to check that the optimizer has not already been built, and raise an error otherwise. Otherwise calling this might not have any effect.

Also need to cast to list.

Reply:
Ok... You have a true. I created a raise an error.

Thanks.